### PR TITLE
switch component creation

### PR DESCRIPTION
**Pull request by Djoudi21**
i extracted the switch component using headless ui library

### CODE REVIEW

**Comment by prevostjules** [comment]
> seems ok for me :) thanks @Djoudi21 Just, after the merge, I think we will have to rename the Colors enum because in staging it is called AliasColors :)

Oh, and I just realized, could you add your .vue component in the ./src/index.ts file so it is exported when we will create the package :)

File: src/components/atomics/ASwitch.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  type: { type: String as PropType<ColorNames>, required: true },
+  label: {
+    type: String,
+    default: ''
+  },
+  checked: {
+    type: Boolean,
+    required: true
+  }
+})
```
Comment (prevostjules) [inline src/components/atomics/ASwitch.vue:43]: could you add here some JSDoc so we know, in storybook and in the project that will install the alias framework, what are this props for ? 
If you want an example, you can check this file https://github.com/progressive-identity/alias-ui-framework/blob/staging/src/components/atomics/AButton.vue

JSDoc is presented like that

```
/**
   * The color of the button
   * The possible values are listed in {@link AliasColorNames}
   */
```

**Review by prevostjules — CHANGES_REQUESTED** [review]

File: src/components/atomics/ASwitch.vue (hunk 1 above)
Comment (Djoudi21) [reply src/components/atomics/ASwitch.vue:43]: thanks Jules for ur feedbacks, i added the comments for the props and i added the component to the index.ts file. sorry for the oversight.

**Review by prevostjules — APPROVED** [review]
seems perfect @Djoudi21 ;)

**Pull request merged**